### PR TITLE
hcloud: Add version 1.21.1

### PR DESCRIPTION
**Pull request by danielwagn3r**
Hetzner Cloud Command-line interface

GitHub: https://github.com/hetznercloud/cli
Documentation: https://developers.hetzner.com/cloud/

### CODE REVIEW

File: bucket/hcloud.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.21.0",
+    "description": "hcloud: Command-line interface for Hetzner Cloud",
```
Comment (earnestma) [inline bucket/hcloud.json:3]:
```suggestion
    "description": "Command-line interface for Hetzner Cloud",

```

File: bucket/hcloud.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.21.0",
+    "description": "hcloud: Command-line interface for Hetzner Cloud",
+    "homepage": "https://github.com/hetznercloud/cli",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/hetznercloud/cli/blob/master/LICENSE"
+    },
```
Comment (earnestma) [inline bucket/hcloud.json:8]: When it's a license identified by SPDX no link is needed

```suggestion
    "license": "MIT",
```

File: bucket/hcloud.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/hetznercloud/cli/releases/download/v$version/hcloud-windows-amd64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/hetznercloud/cli/releases/download/v$version/hcloud-windows-386.zip"
+            }
+        }
+    }
```
Comment (earnestma) [inline bucket/hcloud.json:30]: Autoupdate hash url

```suggestion
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
    }
```

**Review by earnestma — CHANGES_REQUESTED** [review]

**Review by earnestma — COMMENTED** [review]
Please edit PR title to match format: `hcloud: Add version 1.21.0`

File: bucket/hcloud.json (hunk 3 above)
Comment (danielwagn3r) [reply bucket/hcloud.json]: Thank you for the review and your suggestions. I've included them in the PR now and tested it, too.

**Review by r15ch13 — APPROVED** [review]

**Pull request merged**